lib/facter/apt_list_sources.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+def list_sources
```

Comment:
This method should be inlined into the `setcode` block. I don't recall the full explanation of _why_, but [docs mention it](https://puppet.com/docs/puppet/7/fact_overview.html#writing_facts_simple_resolutions-how-to-format-facts). See [internal discussion](https://puppet.slack.com/archives/C02PCQGE24T/p1660687438918419) for more information.

Reply:
Amazing, thank you. Makes total sense.

Reply:
OK thats done!